pkg/operator/v1alpha1helpers/helpers.go

```diff
@@ -207,3 +210,38 @@ func EnsureOperatorConfigExists(client dynamic.Interface, operatorConfigBytes []
 		}
 	}
 }
+
+type InstallConfigGetter func() (installertypes.InstallConfig, error)
+
+// InstallConfigFromFile gets the install config from the cluster configmap file data
+func InstallConfigFromFile(configMapData []byte) InstallConfigGetter {
```

Comment:
is this function really worth to be exposed? In an operator you will most probably use a typed ConfigMap client instead of moving around `[]byte`.

Reply:
This is for reading the ConfigMap from a file, can we do that with a client?

Reply:
Won't you have the install config as yaml directly in a file, not the config map?

Reply:
No, this is intending to use the cluster configmap (which has the install config and network config as nested data) which is already available on the system and generated here in the installer: https://github.com/openshift/installer/blob/master/pkg/asset/manifests/operators.go#L66

We could write the install config to be written as its own file that can be read directly, but imo that seems redundant when we have the data already in the form of the configmap which is going to be using it anyway.